distribution/lambda/Makefile

```diff
@@ -3,7 +3,8 @@
 SHELL := bash
 .SHELLFLAGS := -eu -o pipefail -c
 
-QW_LAMBDA_VERSION?=beta
+# Update this when cutting a new release
+QW_LAMBDA_VERSION?=beta-01
```

Comment:
why not use info from the tag? `lambda-beta-01` for example?

Reply:
or maybe I don't understand how this is used.

Reply:
you mean
```
QW_LAMBDA_VERSION?=$(shell git tag --sort=committerdate | grep -E "lambda-beta-" | tail -1 | cut -b 8-14)
```
?